chaincode/testtuple.go

```diff
@@ -159,6 +159,8 @@ func (testtuple *Testtuple) SetFromTraintuple(db *LedgerDB, traintupleKey string
 		return errors.BadRequest(
 			"could not register this testtuple, the traintuple %s has a failed status",
 			traintupleKey)
```

Comment:
That has nothing to do with this PR per say but we are updating the different train like tuples to be created in a `StatusFailed` if their inModels are failed. Do we want a similar approach for testtuples? (Here we return an error, we could create the testtuple in a failed state)